verilog/tools/ls/vscode/src/download-ls.ts

```diff
@@ -0,0 +1,108 @@
+import { platform, arch } from "os";
+import * as fs from "fs";
+import * as path from "path";
+import { IncomingMessage } from "http";
+import { https as httpsFr } from "follow-redirects";
+import { execSync } from "child_process";
+import * as decompress from "decompress";
+const decompressTargz = require("decompress-targz");
+const TAG = require("../package.json").repository.tag;
+
+
+function checkIfBinaryExists(binaryPath: string) {
+  let whichCommand: string, binaryExists: boolean;
+  if (platform() == "win32") whichCommand = "where";
+  else whichCommand = "which";
```

Comment:
I think these days it is recommended to use `command -v`
e.g. debian warns about `which` being deprecated, on other systems you have to explicitly install `which`; while `command -v` should work with all typical bourne-shells.